XEVIOUS/GameScene.h

```diff
@@ -10,8 +12,18 @@ class GameScene : public Scene
 	~GameScene();
 	void Update();
 	void Render();
+	void KeyOperation();
+	void LoadResources();
 
 private:
-
-
+	DirectX * m_pDirectX;
+	GamePlayer * m_pGamePlayer;
+	GameBullet* m_pGameBullet;
+	CustomVertex m_BG[4];
+	struct InfoGameBG
+	{
+		float CenterBGPosX = 0.f;
+		float CenterBGPosY = 0.f;
+	};
+	void BGScroll(CustomVertex BG[4]);
```

Comment:
「背景をスクロールさせる」であれば
```suggestion
	void ScrollBackground(CustomVertex BG[4]);
```
かな。